apps/remote_control/lib/lexical/remote_control/code_mod/ast/aliases.ex

```diff
@@ -298,4 +289,32 @@ defmodule Lexical.RemoteControl.CodeMod.Ast.Aliases do
   defp collect(elem, %Reducer{} = reducer) do
     {elem, Reducer.update(reducer, elem)}
   end
+
+  defp safe_container_cursor_to_quoted(doc, position) do
+    # https://github.com/elixir-lang/elixir/issues/12673#issuecomment-1592845875
+    # Note: because of the above issue: Using `cursor_context` + `container_cursor_to_quoted`
+    # can't deal with some cases like: `alias Foo.Bar, as: AnotherBar`,
+    # so we need to add a new line to make sure we can get the parrent node of the cursor
+    %{line: line} = position
+    added_new_line_position = Position.new(line + 1, 1)
+    document_fragment = Document.fragment(doc, added_new_line_position)
+
+    case container_cursor_to_quoted(document_fragment) do
+      {:ok, quoted} ->
+        {:ok, quoted}
+
+      _error ->
+        # NOTE: Adding new line not always works,
+        # so we need to try again without adding new line
+        document_fragment = Document.fragment(doc, position)
+        container_cursor_to_quoted(document_fragment)
+    end
+  end
+
+  def container_cursor_to_quoted(document_fragment) do
```

Comment:
why's this public? Shouldn't we always use `safe_container_cursor_to_quoted`? If so, just call the other one `container_cursor_to_quoted`